tests/basic/tests.py

```diff
@@ -574,6 +574,7 @@ class ManagerTest(SimpleTestCase):
         'filter',
         'aggregate',
         'annotate',
+        'alias',
         'complex_filter',
         'exclude',
         'in_bulk',
```

Comment:
We could also add `alias` to the `queries.test_qs_combinators.QuerySetSetOperationTests.test_unsupported_operations_on_combined_qs()`.